config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "appid": "",
   "group": "",
-  "webscene": "19faa71a3bf6468cae35b4fce9393a7d",
+  "webscene": "",
```

Comment:
I guess. But when someone loads the app without any URL param its going to fail. :<